ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -275,34 +275,48 @@ Wait For DSCInitialization CustomResource To Be Ready
 
 Apply DataScienceCluster CustomResource
     [Documentation]
-    [Arguments]        ${dsc_name}=${DSC_NAME}
+    [Arguments]        ${dsc_name}=${DSC_NAME}      ${custom}='False'       ${custom_cmp}=''
```

Comment:
The default value is set to `'False` (with quotes) which is different from `False` (without quotes). It is still working because both cases are different from `True` which is used in the next lines.